README.md

```diff
@@ -152,18 +115,6 @@ my $vars = {
 };
 ```
 
-### Prefer vertically aligned assignments
```

Comment:
+1 on this - 'at your discretion' is an important thing to allow in a style guide